src/main/java/dan200/computercraft/core/apis/OSAPI.java

```diff
@@ -171,12 +171,18 @@ public final void queueEvent( String name, IArguments args )
 
     /**
      * Starts a timer that will run for the specified number of seconds. Once
-     * the timer fires, a timer event will be added to the queue with the ID
-     * returned from this function as the first parameter.
+     * the timer fires, a {@code timer} event will be added to the queue with
+     * the ID returned from this function as the first parameter.
+     *
+     * As with @{os.sleep|sleep}, {@code timer} will automatically be rounded up
+     * to the nearest multiple of 0.05 seconds, as it waits for a fixed amount
+     * of world ticks.
      *
      * @param timer The number of seconds until the timer fires.
-     * @return The ID of the new timer.
+     * @return The ID of the new timer. This can be used to filter the
+     *   {@code timer} event, or @{os.cancelTimer|cancel the timer}.
```

Comment:
```suggestion
     *   {@code timer} event, or {@link #cancelTimer cancel the timer}.
```
Haven't tested, but _should_ work. Won't work with `sleep` for obvious reasons.